### DISCUSSION TITLE
Is 0.694 timesteps/s a normal speed?

### DISCUSSION THREAD

**Discussion by link89** [ACEsuit/mace#441] [Q&A] [answered]
It looks like the lammps project is not allowed to create issue.

I have compile a lammps-mace for CUDA environment and use it to test a 480 atoms system.  The logs show that it does run on GPU but  the speed is relative slow. I am just wondering is it a normal speed or not.

Here is the script of compiling mice: https://gist.github.com/link89/efcad575fb29fe228ace5c417113c957

And the log is in the attachment.
[mace.log](https://github.com/ACEsuit/mace/files/15482376/mace.log)

**Comment by ilyes319** [comment, marked as answer]
Hey @link89, what is your GPU, and what is your model size (number of channels, and equivariance of the model). This is quite slow for a typical mace model.

**Reply by link89** [reply]
It's A100 GPU and after finetune the CPU and threads the speed can reach to 7.974 timesteps/s

**Reply by Juo-kim** [reply]
Hi @link89,
Did you mean that the speed increased after the fine-tuning? 
Could you explain more about that?
I have a similar problem too.
